.gitignore

```diff
@@ -22,5 +22,6 @@ compiled/
 
 # ignore model folder for now
 models
-
-
+# ignore boehm directory that gets generated
+src/backend-c/runtime/boehm-gc
+src/backend-c/runtime/boehm-gc/libatomic_ops
```

Comment:
I am confused, why this was added and deleted in previous commits, then added again in this commit?